comms/dht/src/store_forward/store.rs

```diff
@@ -465,41 +465,7 @@ mod test {
         assert_eq!(messages.len(), 0);
     }
 
-    #[ignore]
-    #[tokio_macros::test_basic]
-    async fn cleartext_join_message() {
```

Comment:
@sdbondi perhaps you know why this was ignored?

Reply:
SAF does not use join messages anymore.

Reply:
This is correct, I ignored this in case we want to bring that functionality back. However I don't think we ever will.